portacle-sly.el

```diff
@@ -0,0 +1,28 @@
+(provide 'portacle-sly)
+
+(remove-hook 'lisp-mode-hook 'slime-lisp-mode-hook)
```

Comment:
Why is this here if it's also done in the advice?

Reply:
Forgot it. Won't hurt anyways, but I'll remove it.

Reply:
Never mind. It is necessary. I added a comment explaining why.